### DISCUSSION TITLE
Feature request: private discussions within organization-wide discussion page

### DISCUSSION THREAD

**Discussion by PaulRBerg** [community/community#59739] [Discussions]
### Select Topic Area

Product Feedback

### Body

We're big fans of the Discussions product, but we often bump into the following circumstance:

We'd like to create a Discussion that remains private to our organization, i.e., which can be seen only by members in a certain [team](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams) that is part of our organization.

**Comment by Daeraxa** [comment]
I think you are asking for something they just removed recently - https://github.blog/changelog/2023-02-08-sunset-notice-team-discussions/ and I'm not sure there is a drop-in replacement for it yet.

Also see https://github.com/orgs/community/discussions/46769

I'd personally like to see access controls on sections and categories that would allow visibility and participation based on the organisation and member structure.

**Reply by mkurz** [reply]
> I'd personally like to see access controls on sections and categories that would allow visibility and participation based on the organisation and member structure.

I think something like this would be the best solution. Allowing making a category/section private, and being able to choose organization members + teams who have permissions to post or view the discussion(s) posted in such a category/section.
